actions/ctf-build-image/action.yml

```diff
@@ -165,7 +165,7 @@ runs:
 
     - name: Build push docker image
       id: build-image
-      uses: smartcontractkit/.github/actions/build-push-docker@build-push-docker/v1
```

Comment:
Keep this pinned to the mutable tag. If this is for testing only, mark as draft please.